readthedocs/core/static-src/core/js/doc-embed/sponsorship.js

```diff
@@ -143,6 +143,27 @@ function adblock_admonition() {
     console.log('--------------------------------------------------------------------------------------');
 }
 
+function adblock_nag() {
+    // Place an ad block nag into the sidebar
+    var placement = create_sidebar_placement();
+    var url = 'https://ads-for-open-source.readthedocs.io/';
```

Comment:
I feel like this should probably go to a landing page that talks about:

* Gold subscriptions
* One time donations
* Ad unblocking
* Our general approach

I don't really know what % of folks will convert, but if we can get the idea of subscriptions removing ads along with the ability to allow us in their ad blocker, it would be good.

Reply:
I think that's a good idea. Conversion percentage is going to be low but we should at least lay out the options. With that said, I don't think creating a landing page for that should hold this PR up.

Reply:
We don't need a landing page, but it should at least mention the ability to pay us, instead of just unblocking the ads.

Reply:
So maybe a "unblock us" or "go ad free with a subscription" could link to both, as an option.

Reply:
Sounds good.

The more we promote going ad-free, the more we need a solution where users can stay logged in for months at a time at their option. Currently login cookies expire after 2 weeks.